### PR TITLE
Tox and pylint integration

### PR DESCRIPTION
**Pull request by stopal-r7**
Tox and pylint integration. tox.ini and .pylinrc files added.
Linting applied to the code.
lecli.py module renamed to cli.py.
load_config moved into functions for unit testing.

### CODE REVIEW

**Comment by stopal-r7** [comment]
@StephenHynes7 @scawley-r7

File: lecli/apiutils.py [hunk 1]
```diff
@@ -1,26 +1,33 @@
...
 
 def load_config():
-  files_read = config.read(configfile)
-  if len(files_read) != 1:
-    print "Error: Config file '%s' not found" % configfile
-    exit(1)
-  if not config.has_section('Auth'):
-    print "Error: Config file '%s' is missing Auth section" % configfile
-    exit(1)
+    config_file = user_config_dir('lecli') + '/' + 'config.ini'
```
Comment (scawley-r7) [inline lecli/apiutils.py]: os.path.join(user_config_dir('lecli'), 'config.ini')
to make it platform independant

File: lecli/apiutils.py [hunk 2]
```diff
@@ -77,37 +100,54 @@ def get_account_resource_id():
...
+    if name in groups:
+        logkeys = filter(None, str(groups[name]).splitlines())
         for logkey in logkeys:
             if len(logkey) != 36:
                 print 'Error: Logkey is not of correct length.'
                 return
         return logkeys
     else:
-        print 'Error: No group with name ' + '\'' + group + '\''
+        print 'Error: No group with name ' + '\'' + name + '\''
```
Comment (scawley-r7) [inline lecli/apiutils.py]: print "Error: No group with name '%s'" % name maybe a bit clearer

Comment (stopal-r7) [inline lecli/apiutils.py]: These are old code - not related to this PR but yeah its much clearer.

File: lecli/apiutils.py [hunk 3]
```diff
@@ -1,26 +1,34 @@
...
+import datetime
+from appdirs import user_config_dir
 
+CONFIG = ConfigParser.ConfigParser()
 
-configfile = 'config.ini'
-config = ConfigParser.ConfigParser()
 
 def load_config():
-  files_read = config.read(configfile)
```
Comment (scawley-r7) [inline lecli/apiutils.py:12]: I'm inclined to think this should live in the application module (lecli.py)
Is config an api utility or is it application config?

**Comment by scawley-r7** [comment]
+1

**Pull request merged**